app/src/main/res/values/strings.xml

```diff
@@ -1180,6 +1180,10 @@
     <string name="search_add_custom_engine_search_string_example">Replace query with “%s”. Example:\nhttps://www.google.com/search?q=%s</string>
     <!-- Text for the button to learn more about adding a custom search engine -->
     <string name="search_add_custom_engine_learn_more_label">Learn More</string>
+    <!-- Accessibility description for the form in which details about the custom search engine are entered -->
+    <string name="search_add_custom_engine_form_description">Custom search engine details</string>
+    <!-- Accessibility description for the 'Learn more' link -->
+    <string name="search_add_custom_engine_learn_more_description">Learn more link</string>
```

Comment:
At risk of asking a very dumb question: why do we need a separate accessibility description for a textual link? Isn't the text of the link normally used in these cases?

Reply:
You can find out my reasoning for this in [my comment](https://github.com/mozilla-mobile/fenix/issues/7967#issuecomment-587522086). The specifications said that the accessibility should announce "Learn more Link", but we can't do that without the description because there is no Type announcement for "link".

Reply:
Thanks, that makes totally sense 👍